reflex-app/src/scenes/Home/Home.js

```diff
@@ -56,9 +65,28 @@ const CTA = styled.button`
   border-radius: 3px;
 `;
 
-const AnimationContainer = styled.div`
+
+const ContentContainer = styled.div`
+  display: grid; 
+  grid-template-columns: 1fr 4fr;
+  grid-template-areas: "m c c c" "m c c c" "m c c c";
+  grid-gap: 15px 20px;
+`
+
+const TableOfContents = styled.div`
+  height: 100vh; // ???
```

Comment:
@gracesnoh IDK what I'm doing here

Reply:
![image](https://user-images.githubusercontent.com/5432430/55930433-023a1280-5bd6-11e9-97ba-97e24e09b81a.png)

Reply:
@lilshim I refactored some stuff and moved the `.selected` style into styled components. The link change behavior seems a little slow to me though, I'll be viewing an animation on the right but it won't reflect the correct one on the left.
<img width="1392" alt="Screen Shot 2019-04-14 at 1 40 11 PM" src="https://user-images.githubusercontent.com/8237984/56098902-d3b08600-5eba-11e9-951c-a7f8d8b9c0f1.png">

Reply:
Yea I notice that too... I'll take a stab at fixing